web/src/components/home/subcomponents/upcoming_events/messages.js

```diff
@@ -0,0 +1,13 @@
+const messages = {
+  atStudioM: "@ Studio M Ballroom, San Jose, CA",
+  viewOnFacebook: "View on Facebook",
+  dyosEntryFee: "Entry fee: $10-$40",
+  dyosMaskingRequirement: "Masks required during classes",
+  upcomingEventsTitle: "Upcoming events",
+  upcomingEventsDescription:
+    "See the Classes tab for class description, prerequisites, pricing, and masking policy.",
```

Comment:
This is going to include a link, I don't think we want this as a message since `Classes` will need to be formatted & styled independently

Reply:
Yeah I initially wrote a TODO, then I decided we can figure it out later. It's not going to be visible in prod yet, so I say we should just keep it for now, then figure out a better solution later